### PR TITLE
93 contacts

### PR DESCRIPTION
**Pull request by tbaxter-18f**
First pass at models (reasonably complete) with scrubbed data for the new model. Needs admin work.

### CODE REVIEW

File: nrm_django/contacts/admin.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from .models import Contact
+
+
+class ContactAdmin(admin.ModelAdmin):
+    pass
```
Comment (tbaxter-18f) [inline nrm_django/contacts/admin.py:7]: Just did the bare minimum in admin for now, thinking that when we pick up #156 we'll have a better idea of what should be visible, inline, or just hidden.

File: nrm_django/grants/migrations/0009_auto_20210329_1606.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.1.7 on 2021-03-29 16:06
```
Comment (neilmb) [inline nrm_django/grants/migrations/0009_auto_20210329_1606.py:1]: I there something strange about this merge? Do we need to merge main back into this? I didn't expect there to be any changes to these migrations from the contacts change.

File: nrm_django/grants/admin.py [hunk 3]
```diff
@@ -16,9 +16,9 @@ class GrantAuthorityInline(admin.TabularInline):
 class GrantAdmin(admin.ModelAdmin):
     inlines = [GrantAuthorityInline, NoteInline]
     list_display = (
-        "pretty_name",
         "gid",
         "pretty_cooperator_name",
+        "pretty_name",
```
Comment (neilmb) [inline nrm_django/grants/admin.py:21]: Same concern, changing `GrantAdmin` seems unrelated to the contacts changes?

**Review by neilmb — APPROVED** [review]
Looks good, but it seems like it incorporates two changes from a different PR on main related to grants display?

File: nrm_django/grants/admin.py (hunk 3 above)
Comment (tbaxter-18f) [reply nrm_django/grants/admin.py:21]: I was sneaking a small admin change in based on last week's feedback.

File: nrm_django/grants/migrations/0009_auto_20210329_1606.py (hunk 2 above)
Comment (tbaxter-18f) [reply nrm_django/grants/migrations/0009_auto_20210329_1606.py:1]: Hmmmm. This may be an unapplied migration from 111. Not sure. Want to yank it out of this commit and bring it in separate?
